client/flutter/pubspec.lock

```diff
@@ -64,6 +64,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.16.1"
+  equatable:
```

Comment:
I can't see Equatable being used anywhere?

Reply:
It is a transitive dep.  pubspec.lock is auto generated from pubspec.yaml